resolvers.ts

```diff
@@ -40,6 +40,31 @@ export const resolvers: Resolvers = {
         };
       }, []);
     },
+    BulkDownloadCGOverview: async (root, args, context, info) => {
+      const body = {
+        download_type: args?.input?.downloadType,
+        workflow: args?.input?.workflow,
+        params: {
+          include_metadata: { value: args?.input?.includeMetadata }, 
+        sample_ids: {
+          value: args?.input?.workflowRunIds
+        }, 
+        workflow: {
+          value: "consensus_genome"
+        }
+        },
+        workflow_run_ids: args?.input?.workflowRunIds,
```

Comment:
nit: could we destructure `args?.input` to make this a bit more succinct?  Also, it seems like if `args.input` (or any of the fields) do not exist, would we want to make the API call even?